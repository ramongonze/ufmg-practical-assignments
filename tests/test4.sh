#!/bin/bash
set -u

i=4

gcc -g -Wall -I. tests/test$i.c fs.o -o test$i &>> gcc.log
if [ ! -x test$i ] ; then
    echo "[$i] compilation error"
    exit 1 ;
fi

if ! ./test$i > test$i.out 2> test$i.err ; then
    echo "[$i] error"
    exit 1
fi

rm -f test$i test$i.out test$i.err
exit 0
