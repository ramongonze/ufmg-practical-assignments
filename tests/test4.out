parent of test4 with parameter 4
parent of test3 with parameter 3
parent of test2 with parameter 2
parent of test1 with parameter 1
